src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -201,18 +187,6 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
         }
     }
 
-    private getOpenNotebookCount(type: BannerType): number {
-        switch (type) {
-            case BannerType.InsidersNotebookSurvey:
-                return this.getPersistentState(InsidersNotebookSurveyStateKeys.OpenNotebookCount);
```

Comment:
Can we delete this enum value from InsidersNotebookSurveyStateKeys?

Reply:
@joyceerhl Good call. I'll update. Also I made a tweak to the interface that we are using. It was actually hooked up to an old logger that wasn't even functioning anymore for execution count 😱 .